src/SUMMARY.md

```diff
@@ -1,30 +1,73 @@
-# Summary
+# The Edition Guide
 
 [Introduction](introduction.md)
 
-- [What are Editions?](editions/index.md)
-    - [Transitioning your code to a new edition](editions/transitioning.md)
-
-- [Rust 2015](2015/index.md)
-
-- [Rust 2018](2018/index.md)
-    - [Feature status](2018/status.md)
-    - [Transitioning to Rust 2018](2018/transitioning/to-rust-2018.md)
-        - [Trait system](2018/transitioning/traits/index.md)
-            - [`dyn Trait`](2018/transitioning/traits/dyn-trait.md)
-            - [`impl Trait`](2018/transitioning/traits/impl-trait.md)
-        - [Module system](2018/transitioning/modules/index.md)
-            - [Path clarity](2018/transitioning/modules/path-clarity.md)
-            - [Macro changes](2018/transitioning/modules/macros.md)
-        - [Concurrency additions](2018/transitioning/concurrency/index.md)
-            - [`async`/`await`](2018/transitioning/concurrency/async-await.md)
-        - [Ownership and lifetimes](2018/transitioning/ownership-and-lifetimes/index.md)
-            - [`'_`, the anonymous lifetime](2018/transitioning/ownership-and-lifetimes/anonymous-lifetime.md)
-            - [In-band lifetimes](2018/transitioning/ownership-and-lifetimes/in-band-lifetimes.md)
-            - [Default match bindings](2018/transitioning/ownership-and-lifetimes/default-match-bindings.md)
-            - [Lifetime elision in `impl`](2018/transitioning/ownership-and-lifetimes/lifetime-elision-in-impl.md)
-            - [`T: 'a` inference in structs](2018/transitioning/ownership-and-lifetimes/struct-inference.md)
-        - [Slice patterns](2018/transitioning/slice-patterns.md)
-        - [Error Handling](2018/transitioning/errors/index.md)
-            - [`?` in `fn main()` and `#[test]`s](2018/transitioning/errors/question-mark.md)
-        - [Raw identifiers](2018/transitioning/raw-identifiers.md)
+## What are editions?
+
+- [What are editions?](editions/index.md)
+  - [Transitioning your code to a new edition](editions/transitioning-your-code-to-a-new-edition.md)
+
+## Rust 2015
+
+- [Rust 2015](rust-2015/index.md)
+
+## Rust 2018
+
+- [Rust 2018](rust-2018/index.md)
```

Comment:
We should probably organize this a bit into categories after this PR; The flat list is a bit chaotic.

Reply:
I found the categories far more confusing, personally

Reply:
I guess it doesn't work well with ordering also... let's not do it for now then :)